templates/epic.xml.jinja2

```diff
@@ -181,18 +181,26 @@
   <documentation level="11">
     ## Far foward detectors
   </documentation>
-  {% for ff in features['far_forward'] -%}
-  <include ref="${DETECTOR_PATH}/compact/{{ ff }}.xml"/>
-  {% endfor -%}
+  {% if features['far_forward'] | length > 1 -%}
```

Comment:
I don't like the non-uniform approach. Better, just move `compact/far_forward.xml` to `compact/far_forward/full.xml` and keep the explicit `full:` in the configuration yamls.

Reply:
Yeah, jinja doesn't like it either. `full.xml` in the categories is a good idea.

Reply:
I'm going to go with `default` instead of `full` since in some cases (far backward here), the default is to have fewer elements than some other 'extended' implementations.

Reply:
Didn't change the brycecanyon ones removed in #674, but otherwise ready for review again.